calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,234 @@
+```
+
+Once the `Send calibration` button is clicked, the result will be sent to the sensor calibration manager. No pairs can be added or deleted afterward like the image shown below. Please make sure you want to end the calibration process when you click the button.
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/end_calibration1.jpg" alt = "end_calibration1" width = 700px></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/end_calibration2.jpg" alt = "end_calibration2" width = 700px></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Rosbag finished.</p></td>
+    <td><p style="text-align: center;">After clicking send calibration.</p></td>
+  </tr>
+</table>
+
+## Results
+
+After the calibration process finishes, the `sensor_calibration_manager` will display the results in the UI and allow the user to save the calibration data to a file.
+
+In the UI of the X2 project, three different TF trees are displayed: `Initial TF Tree`, `Calibration Tree`, and `Final TF Tree`.
+
+- The `Initial TF Tree` presents the initial TF connections between sensors needed for calibration.
+- The `Calibration Tree` shows the calibrated transformation between sensors, in this tutorial, `front_center/radar_link` to `pandar_40p_front`.
+- The `Final TF Tree` depicts the TF tree after incorporating the updated calibrated transformation. As autoware utilizes the concept of [sensor_kit](https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-model/creating-sensor-model/), the final transformation required to comply to the specifications is `front_unit_base_link` to `front_center/radar_link`. This transformation is performed by the [calibrator interface](../../sensor_calibration_manager/sensor_calibration_manager/calibrators/x2/marker_radar_lidar_calibrator.py) related to this project. The red arrow indicates that the final transformation changed after the calibration process.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu4.jpg" alt="menu4"  width="500">
+</p>
+
+To evaluate the calibration result, the user can measure that the calibrated radar points (green) are closer than the initial radar points (red) to the lidar points (white).
+
+## FAQ
+
+- Why doesn't the reflector detection show on the rviz?
+
+  - Make sure the center of the reflector faces toward the radar sensor, and the height of the reflector is enough for the radar to detect.
+  - Make sure the height of the radar reflector is not larger than the `reflector_max_height` parameter.
+  - Make sure the radar reflector is not in the background voxel (visualize the topic mentioned before.)
+
+- Why doesn't the calibration error seem low enough?
+
+  - Make sure that there are no outliers in the calibration pairs list.
+  - Make sure that the initial calibration is good enough to match the lidar detection and radar detection correctly.
```

Comment:
Radars like the ARS408 (the one from the example) have a resolution of 0.2m. Giving that, there is a theoretical limit of how low the calibration error can be. Point being, if the radar resolution is considerable, it will strongly limit lower bound of the calibration error

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/88184a9371eb63d46723c1e528f6e45b784964e2